java/spring/container/Application/azure-pipelines.yml

```diff
@@ -26,6 +26,13 @@ steps:
       targetFolder: '$(Build.ArtifactStagingDirectory)'
       cleanTargetFolder: true
     displayName: 'Copy docker compose file to staging directory'
+  - task: CopyFiles@2
+    inputs:
+      sourceFolder: $(Build.SourcesDirectory)/armtemplates/docker-for-azure
```

Comment:
`/armtemplates/docker-for-azure` could potentially be an environment variable if it will be used across other things